config/initializers/devise.rb

```diff
@@ -228,4 +228,21 @@
   #   manager.intercept_401 = false
   #   manager.default_strategies(:scope => :user).unshift :some_external_strategy
   # end
-end
+  config.saml_create_user = true
+  config.saml_update_user = true
+  config.saml_default_user_key = :email
+  config.saml_session_index_key = :session_index
+  config.saml_use_subject = true
+  config.idp_settings_adapter = nil
+  config.saml_configure do |settings|
+    settings.assertion_consumer_service_url     = ENV["ASSERTION_CONSUMER_SERVICE_URL"]
+    settings.assertion_consumer_service_binding = "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST"
```

Comment:
What if we need to authenticate both Studio users and Editions users?

Reply:
@dleadbetter this is a question that we can table for now -- if another application needs to authenticate against the Purdue IdP, via HTTP-Redirect through Annotation Studio, we will deal with that in a separate PR.